service/project.go

```diff
@@ -248,7 +243,57 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var hook *model.GithubHook
+	hook, err = model.FindGithubHook(responseRef.Owner, responseRef.Repo)
+	if err != nil {
+		uis.LoggedError(w, r, http.StatusInternalServerError, err)
+		return
+	}
+	origGithubWebhookEnabled := (hook != nil)
+	if hook == nil {
+		var hookID int
+		hookID, err = uis.setupGithubHook(responseRef.Owner, responseRef.Repo)
+		if err != nil {
+			grip.Error(message.WrapError(err, message.Fields{
+				"source":  "project edit",
+				"message": "can't setup webhook",
+				"project": id,
+				"owner":   responseRef.Owner,
+				"repo":    responseRef.Repo,
+			}))
+			// don't kill it here, sometimes people change
+			// Evergreen to track a personal branch,
+			// one that we have no access to
+			projectRef.TracksPushEvents = false
+		} else {
+			hook := model.GithubHook{
+				HookID: hookID,
+				Owner:  responseRef.Owner,
+				Repo:   responseRef.Repo,
+			}
+			if err = hook.Insert(); err != nil {
+				// A github hook as been created, but we couldn't
+				// save the hook ID in our database. This needs
+				// manual attention for clean up
+				grip.Alert(message.WrapError(err, message.Fields{
```

Comment:
This sends us a slack message. I think it's better to indicate to the user that there's a problem and let them reach out to us.

Reply:
Made it more clear they should reach out to us:
![screen shot 2019-02-28 at 4 18 26 pm](https://user-images.githubusercontent.com/17027265/53599216-aa3ed380-3b74-11e9-809b-accea52db30c.png)

Reply:
nit: "cannot" instead of "can not"